agent/06_agent_create_cluster.sh

```diff
@@ -14,6 +14,12 @@ source $SCRIPTDIR/agent/common.sh
 
 early_deploy_validation
 
+function create_pxe_files() {
+    local asset_dir="${1:-${OCP_DIR}}"
+    local openshift_install="$(realpath "${OCP_DIR}/openshift-install")"
+    "${openshift_install}" --dir="${asset_dir}" --log-level=debug agent create pxe-files
```

Comment:
nit: there's some duplication here that could be removed by making parametric the agent subcommand params